test/unit/high-security.test.js

```diff
@@ -41,238 +35,278 @@ function getPath(obj, path) {
   return obj[paths[0]]
 }
 
-describe('high security mode', function () {
-  describe('config to be sent during connect', function () {
+tap.Test.prototype.addAssert('check', 3, function (key, before, after) {
+  var fromFile = { high_security: true }
+  setPath(fromFile, key, before)
+
+  var config = new Config(fromFile)
+  return this.same(getPath(config, key), after)
```

Comment:
didn't realize you had to return in these custom assertions, I wonder if the ones I wrote actually work 🤔 . anyways, great job on abtracting this 👏🏻